### ISSUE TITLE
pa11y fails with Github Actions ubuntu-22.04

### ISSUE THREAD

**Issue by bansan85** [pa11y/pa11y#651]
I tried to update CI Github Actions of my project from 'ubuntu-latest' to 'ubuntu-22.04' (beta).

Now, pa11y always runs on timeout.

I tried to update CI Github Actions of pa11y project. As you can see, running tests fails (https://github.com/bansan85/pa11y/runs/7273637127)

**Comment by davidwilby** [comment]
Similar problem on ubuntu 22.04 desktop, reported in #662

**Comment by hollandjg** [comment]
GitHub is currently switching over from ubuntu 20.04 to 22.04 as the `ubuntu-latest` runner, so this issue will become much more prevalent. We had to deal with it too.
The ability to get the debug messages from pa11y (or the log files) would have been super useful when debugging this. The way we worked out what was happening in the end was to add an additional step in the GitHub actions which ran `pa11y` with the `--debug` flag and that showed that ChromeHeadless just wasn't loading.
